### PR TITLE
add ben haller to the author list

### PR DESCRIPTION
**Pull request by petrelharp**
I've been reminded that @bhaller did a bunch of work on SLiM at our request, that has made developing stdpopsim a lot easier, e.g., adding Dictionary( )/DataFrame( ) (which as made unit testing *much* easier), and so (IMO) ought to be in here.

### CODE REVIEW

**Review by andrewkern — APPROVED** [review]

**Comment by andrewkern** [comment]
LGTM

File: adding_species_manuscript.tex [hunk 1]
```diff
@@ -71,35 +71,36 @@
...
+\author[27]{Juan Manuel Vasquez}
+\author[28]{Ryan Waples}
+\author[29]{Anthony Wilder Wohns}
+\author[210]{Yan Wong}
+\author[31]{Reed Cartwright}
+\author[32]{Aaron P. Ragsdale}
+\author[33]{Franz Baumdicker}
+\author[34]{Gregor Gorjanc}
+\author[35]{Ryan N. Gutenkunst}
+\author[210]{Jerome Kelleher}
```
Comment (igronau) [inline adding_species_manuscript.tex]: this should be \author[30], right?

File: adding_species_manuscript.tex [hunk 2]
```diff
@@ -71,35 +71,36 @@
...
-\author[31]{Aaron P. Ragsdale}
-\author[32]{Franz Baumdicker}
-\author[33]{Gregor Gorjanc}
-\author[34]{Ryan N. Gutenkunst}
-\author[29]{Jerome Kelleher}
+\author[26]{Per Unneberg}
+\author[27]{Juan Manuel Vasquez}
+\author[28]{Ryan Waples}
+\author[29]{Anthony Wilder Wohns}
+\author[210]{Yan Wong}
```
Comment (igronau) [inline adding_species_manuscript.tex]: this should be \author[30], right?

**Review by igronau — APPROVED** [review]
see two minor typos in my comments

File: adding_species_manuscript.tex (hunk 2 above)
Comment (petrelharp) [reply adding_species_manuscript.tex]: ack what happened there

Comment (petrelharp) [reply adding_species_manuscript.tex]: oh I know what happened, heh - I asked vim to increment all the numbers but had only the last digit highlighted. =)

**Pull request merged**